site/content/docs/commands/secret-init.en.md

```diff
@@ -0,0 +1,93 @@
+# secret init
+```
+$ copilot secret init
+```
+
+## What does it do?
+`copilot secret init` creates or updates secrets as [SecureString parameters](https://docs.aws.amazon.com/systems-manager/latest/userguide/systems-manager-parameter-store.html#what-is-a-parameter) in SSM Parameter Store for your application.
+
+A secret can have different values in each of your existing environments, and is accessible by your services or jobs from the same application and environment.
+
+!!! attention 
+    Secrets are not supported for Request-Driven Web Services.
+
+## What are the flags?
+```
+  -a, --app string              Name of the application.
+      --cli-input-yaml string   Optional. A YAML file in which the secret values are specified.
+                                Mutually exclusive with the -n, --name and --values flags
+  -h, --help                    help for init
+  -n, --name string             The name of the secret.
+                                Mutually exclusive with --cli-input-yaml flag.
```

Comment:
```suggestion
                                Mutually exclusive with the --cli-input-yaml flag.
```
I don't really understand why the period use is consistently inconsistent in our help menus. (The "mutually exclusive" lines don't have periods, but others, excluding the always-lowercase "help for..." do.) Let's just keep the period here and add it to the others!

Reply:
added to all other flag desc for all commands and updated their site docs accordingly